### PR TITLE
Add my PGP fingerprint

### PR DESCRIPTION
**Pull request by joshuagl**
Please fill in the fields below to submit a pull request.  The more information
that is provided, the better.

**Fixes issue #**: N/A

**Description of the changes being introduced by the pull request**:
* Add my PGP fingerprint to the MAINTAINERS file
* ~Fix a typo in the PULL_REQUEST_TEMPLATE (fulfills -> fulfils)~

**Please verify and check that the pull request fulfils the following
requirements**:

- [ ] The code follows the [Code Style Guidelines](https://github.com/secure-systems-lab/code-style-guidelines#code-style-guidelines)
- [ ] Tests have been added for the bug fix or new feature
- [ ] Docs have been added for the bug fix or new feature

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ that is provided, the better.
 
 **Description of the changes being introduced by the pull request**:
 
-**Please verify and check that the pull request fulfills the following
+**Please verify and check that the pull request fulfils the following
```
Comment (trishankatdatadog) [inline .github/PULL_REQUEST_TEMPLATE.md]: I dunno, this seems to be a British vs American English thing... 😅

File: docs/MAINTAINERS.txt [hunk 2]
```diff
@@ -38,3 +38,4 @@ Maintainers:
   Joshua Lock
     Email: jlock@vmware.com
     GitHub username: @joshuagl
+    PGP fingerprint: 08F3 409F CF71 D87E 30FB D3C2 1671 F65C B748 32A4
```
Comment (trishankatdatadog) [inline docs/MAINTAINERS.txt:41]: Do you have a Keybase profile we can use to verify?

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (joshuagl) [reply .github/PULL_REQUEST_TEMPLATE.md]: Oh no! 😬

File: docs/MAINTAINERS.txt (hunk 2 above)
Comment (joshuagl) [reply docs/MAINTAINERS.txt:41]: I've just signed up, verified in a couple of places and updated this patch to add the username.

**Review by trishankatdatadog — APPROVED** [review]

**Comment by trishankatdatadog** [comment]
Verified on https://keybase.io/joshuagl

**Pull request merged**